db/migrate/20161010151307_create_physical_servers.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePhysicalServers < ActiveRecord::Migration[5.0]
+  def change
+    create_table :physical_servers do |t|
+      t.bigint :ems_id
+      t.string :name
+      t.string :type
+      t.string :uuid
```

Comment:
The `uuid` into `uid_ems`? I got `uuid` from @blomquisg's document, but I think I see what you're saying. This is a case where we'd turn a providers `uuid` into a generic `MiQ uuid`?